<launch>

  <node pkg="pr2_tuckarm" type="tuck_arms.py" name="tuck_arms_app" />
  <node pkg="pr2_position_scripts" type="head_up.py" name="head_up" />
  <node pkg="pr2_position_scripts" type="torso_down.py" name="torso_down" />

  <!-- Action server for sensing motion of the grippers during pickup and dropoff. -->
  <include file="$(find pr2_gripper_sensor_action)/launch/pr2_gripper_sensor_actions.launch" />

  <!-- Navigation -->
  <include file="$(find pr2_delivery)/launch/pr2_delivery_nav/nav.launch" />
  
  <node pkg="map_store" type="map_manager" name="map_manager" output="screen"/>
  
  <node name="council_demo" pkg="bmce_apps" type="council_demo.py" output="screen">
    <param name="data_dir" value="$(find bmce_apps)/cfg/council" />
    <rosparam command="load" file="$(find bmce_apps)/cfg/council/delivery_phrases.yaml" />
  </node>
  
</launch>
